### PR TITLE
updated dsa section 

### PR DESCRIPTION
**Pull request by Sushilverma002**
close issue -> #1337

## Screenshots

image

### CODE REVIEW

**Comment by Sushilverma002** [comment]
@CBID2 please review.

File: database/youtube/data-structures.json [hunk 1]
```diff
@@ -105,10 +105,18 @@
   },
   {
     "name": "TAP ACADEMY",
-    "description": "A well-structured DSA playlist from beginner to advanced level in Java",
+    "description": "A well-structured DSA playlist from beginner to advanced level in Java.",
```
Comment (CBID2) [inline database/youtube/data-structures.json:108]: [Proofreading]
```suggestion
    "description": "A well-structured DSA playlist that provides beginner to advanced level tutorials in Java.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Found a typo but you're off to a good start

File: database/youtube/data-structures.json (hunk 1 above)
Comment (Sushilverma002) [reply database/youtube/data-structures.json]: updated @CBID2

**Review by CBID2 — APPROVED** [review]
LGTM! 😊

**Comment by Sushilverma002** [comment]
@CBID2 can u please add level label and gssoc'23

**Review by k-deepak04 — APPROVED** [review]

**Pull request merged**